<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DispatchOrderWidget</class>
 <widget class="QWidget" name="DispatchOrderWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1571</width>
    <height>902</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dispatch Order</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1551</width>
     <height>661</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>721</width>
          <height>571</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ID:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineedit_po_id">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Dispatch Date:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Delivery Date:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>External Reference:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineedit_po_external_reference">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateTimeEdit" name="datetimeedit_dispatch_date"/>
         </item>
         <item row="2" column="1">
          <widget class="QDateTimeEdit" name="datetimeedit_delivery_date">
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="formLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>770</x>
          <y>20</y>
          <width>751</width>
          <height>571</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Shipping Agent:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Shipping Agent Service:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Tracking Code:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineedit_po_tracking_code"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Carrier Reference:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineedit_po_carrier_reference"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="combobox_shipping_agent"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineedit_shipping_agent_service"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Delivery Address</string>
       </attribute>
       <widget class="QWidget" name="formLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>661</width>
          <height>671</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Phone:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Email:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Address:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineedit_po_da_name"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineedit_po_da_phone"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineedit_po_da_email"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineedit_po_da_address"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="formLayoutWidget_5">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>20</y>
          <width>811</width>
          <height>671</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Postcode:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>City:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Country Code:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>Country:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineedit_po_da_city"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineedit_po_da_postcode"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineedit_po_da_countrycode"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineedit_po_da_country"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Lines</string>
       </attribute>
       <widget class="QTableWidget" name="tablewidget_lines">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>1501</width>
          <height>561</height>
         </rect>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <property name="gridStyle">
         <enum>Qt::SolidLine</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>25</number>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>680</y>
     <width>861</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btn_close">
      <property name="styleSheet">
       <string notr="true">min-height:1.5em;</string>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_dispatch">
      <property name="styleSheet">
       <string notr="true">min-height:1.5em;</string>
      </property>
      <property name="text">
       <string>Dispatch</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
